packages/security/carbon.yml

```diff
@@ -0,0 +1,311 @@
+      framework: react
+      thumbnailPath: './thumbnails/string-formatter.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-components-stringformatter
+      tags:
+        - content-element
+  time-indicator:
+      name: Time indicator
+      status: stable
+      type: component
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/time-indicator.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-components-timeindicator
+      tags:
+        - system-feedback
+  truncated-list:
+      name: Truncated list
+      status: stable
+      type: component
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/truncated-list.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-components-truncatedlist
+      tags:
+        - data-display
+        - content-element
+  type-layout:
+      name: Type layout
+      status: stable
+      type: component
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/input.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-components-typelayout
+      tags:
+        - data-display
+        - content-element
+  combo-button:
+      name: Combo button
+      status: stable
+      type: pattern
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/combo-button.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-patterns-combobutton
+      tags:
+        - input-control
+  filter-panel:
+      name: Filter panel
+      status: stable
+      type: pattern
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/filter-panel.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-patterns-filterpanel
+      tags:
+        - input-control
+        - data-display
+  icon-button-bar:
+      name: Icon button bar
+      status: stable
+      type: pattern
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/icon-button-bar.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-patterns-iconbuttonbar
+      tags:
+        - input-control
+        - data-display
```

Comment:
Delete data-display (I kept it for the above component because it displays the number of filtered results, but I didn't see any data display in this one)